pkgs/top-level/python-packages.nix

```diff
@@ -5293,6 +5293,8 @@ in {
 
   pluggy = callPackage ../development/python-modules/pluggy {};
 
+  pluggy_0_8 = callPackage ../development/python-modules/pluggy/0_8.nix {};
```

Comment:
This one will have to always be used with Python 2. Note though, if we notice breakage due to this change, then that means we won't be able to support pytest 3 anymore.

Reply:
I tried, but it breaks 
+ `python3Packages.pytest_3` it would use pluggy=0.12.0, which was the original bug report
+ `python{2,3}Packages.pytest` because it requires [`pluggy>=0.12`](https://github.com/pytest-dev/pytest/blob/4.6.3/setup.py#L17)
>  then that means we won't be able to support pytest 3 anymore.

I'm inclined to drop pytest_3, only 9 packages depend on it